model/job.js

```diff
@@ -12,9 +9,15 @@ const MODEL = {
         .example('50dc14f719cdc2c9cb1fb0e49dd2acc4cf6189a0'),
 
     name: Joi
-        .string()
+        .string().regex(/^[A-Za-z0-9_-]+$/).min(3)
```

Comment:
I was thinking more around how domain names are minimum lengthed.  Anything under 3 doesn't provide enough context.

Actually, domain names minimum is 2 characters: https://www.quora.com/What-is-the-minimum-number-of-characters-for-a-domain-name

Do you think this is unnecessary?